pyntc/devices/ios_device.py

```diff
@@ -42,12 +42,11 @@ def __init__(self, host, username, password, secret="", port=22, **kwargs):
         self.open()
 
     def _enable(self):
-        self.native.exit_config_mode()
```

Comment:
Since this exact API is duplicated across multiple devices, should it be abstracted to a Superclass?

Reply:
I do want to revisit the design, but not at this time; I want to focus on making sure everything works and tests are working again.